app/assets/stylesheets/local/show.scss

```diff
@@ -1,29 +1,165 @@
-// There seems to be no existing css class allowing us to scope
-// this stuff just to generic work 'show' page. If there was, I'd use it.
-// Perhaps it should be added later.
 
-.additional-titles {
-  h2 {
-    @extend h4;
+
+.show-page-image {
+  position: relative;
+
+  .show-page-image-image {
+    width: 100%;
+    // keep crazy long aspect ratio images from being
+    // so long, although there will be some distortion.
+    max-height: 80vw;
+  }
+}
+
+.show-page-image-viewer-link {
+  display: block;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+.show-page-image-bar {
+  position: absolute;
+  display: flex;
+  width: 100%;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: transparentize($brand-primary, .3);
+  .btn-group {
+    width: 50%;
+    .btn { width: 100%; }
+    .dropdown-menu {
+      left: 5px;
+    }
+  }
+  a, button {
+    flex: 1;
+    border-radius: 0;
+    background: transparent;
   }
 }
 
-.large-rights-statement {
-  margin-top: 15px;
+.show-page-image-permission {
+  .label {
+    position: absolute;
+    right: 0;
+    top: 0;
+    border-top-right-radius: 0;
+    border-top-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+}
+
+
+
+
+.work-show {
+  padding: 0 15px;
+
+  .work-show-admin {
+    text-align: right;
+  }
+
+  .show-page-image.thumb {
+    display: inline-block;
+    width: 160px;
+    margin: 4px 4px 0 0;
+
+    img {
+      width: 100%;
+      &.lazyload, &.lazyloading {
+        background: #AAA asset-url('spinner.svg') no-repeat center;
+        height: calc(1.33 * 160px);
+      }
+      vertical-align: bottom;
+      // keep crazy long aspect ratio images from being
+      // so long, although there will be some distortion.
+      max-height: 30vw;
+      min-height: 60px; // just so it's something!
+    }
+
+    @media (min-width: $screen-sm-min) {
+      // 2 wide
+      width: calc(50% / 2 - 12px);
+      // before lazy loaded complete, VERY roughy estimate 4/3 aspect ratio ish
+      img.lazyload, img.lazyloading {
+        height: 25vw;
+      }
+    }
+    @media (min-width: $screen-lg-min) {
+      // 3 wide
+      width: calc(50% / 3 - 10px);
+      img.lazyload, img.lazyloading {
+        height: 15vw;
+      }
+    }
+  }
+
+  @media (min-width: $grid-float-breakpoint) {
```

Comment:
curious why you use `grid-float-breakpoint` here instead of `screen-sm-min` since they are the same. Never really used the former. Have you seen cases where you need to change one to be different from the other?